.env.appStore.example

```diff
@@ -126,5 +126,16 @@ SALESFORCE_CONSUMER_SECRET=""
 ZOHOCRM_CLIENT_ID=""
 ZOHOCRM_CLIENT_SECRET=""
 
-
+# Used for the Pipedrive CRM integration
+PIPEDRIVE_CLIENT_ID=""
+PIPEDRIVE_CLIENT_SECRET=""
```

Comment:
I don't think we need these. These should be stored in DB and configurable via settings/admin/apps

Reply:
Just tested that the values are available there, so we can remove these safely. Should update the README as well to update the values in admin instead of env variables.
![image](https://github.com/calcom/cal.com/assets/1780212/4a70a41b-d09c-451e-abcd-69be77a654ae)

Reply:
@hariombalhara I'm looking into this. Would you be able to tell me which action triggers this popup? 

Also after the lates pull from `main` I keep getting a `404` on the event-types link. 

Any idea why I might be facing this or how I could resolve this locally? 

<img width="1012" alt="Screenshot 2023-12-19 at 1 59 08 PM" src="https://github.com/calcom/cal.com/assets/7681067/9f806751-0a8e-4335-95d2-d1d98d604902">